svcs/guacamole/event.go

```diff
@@ -95,8 +95,55 @@ type eventHost struct {
 	dir       string
 }
 
-//Create the event configuration for the event got from the DB
-func (eh *eventHost) CreateEventFromEventDB(ctx context.Context, conf store.EventConfig, reCaptchaKey string) (Event, error) {
+func protobufToJson(message proto.Message) (string, error) {
```

Comment:
Why not proto directly to struct?

Reply:
protobufToJson did not added on this PR, it was there, so it would be nice to consider it separately in another discussion.